### PR TITLE
Support weave network plugin

### LINKED ISSUE #216: Pods with weave network plugin are always creating
Pods with weave network plugin are always creating, this is because weave plugin creates a veth pair based on a truncated sandboxID, e.g. vethweplk8s_POD. But for hyper runtime, sandboxID always have same prefix `k8s_POD.`, which results in same veth name `vethweplk8s_POD` in weave plugin.

### PR DESCRIPTION
**Pull request by feiskyer**
Fix #216.

To use weave network plugin, run

```sh
  kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
```

### CODE REVIEW

File: pkg/hyper/sandbox.go [hunk 1]
```diff
@@ -77,15 +77,20 @@ func (h *Runtime) RunPodSandbox(config *kubeapi.PodSandboxConfig) (string, error
 		"portMappings": portMappingsParam,
 	}
 	podId := userpod.Id
-	_, err = h.netPlugin.SetUpPod(netNsPath, podId, config.GetMetadata(), config.GetAnnotations(), capabilities)
+	sandboxID := podId
+	// workaroud for weave netwprk plugin because it creates a veth pair based on a truncated sandboxID.
```
Comment (resouer) [inline pkg/hyper/sandbox.go]: typo: netwprk -> network

File: pkg/hyper/sandbox.go [hunk 2]
```diff
@@ -291,7 +296,12 @@ func (h *Runtime) StopPodSandbox(podSandboxID string) error {
 	}
 
 	// 4: tear down the cni network.
-	err = h.netPlugin.TearDownPod(netNsPath, podSandboxID, status.GetMetadata(), status.GetAnnotations(), capabilities)
+	sandboxID := podSandboxID
+	// workaroud for weave netwprk plugin because it creates a veth pair based on a truncated sandboxID.
```
Comment (resouer) [inline pkg/hyper/sandbox.go]: same typo

**Comment by feiskyer** [comment]
@resouer  Fixed typo. PTAL

**Pull request merged**